src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -175,26 +176,38 @@ class ParseLiveQueryServer {
               if (!isMatched) {
                 return null;
               }
+              aclError = false;
               res = {
                 event: 'Delete',
                 sessionToken: client.sessionToken,
                 object: deletedParseObject,
                 clients: this.clients.size,
                 subscriptions: this.subscriptions.size,
                 useMasterKey: client.hasMasterKey,
-                installationId: client.installationId
+                installationId: client.installationId,
+                sendEvent: true,
               };
               return maybeRunAfterEventTrigger('afterEvent', className, res);
             })
             .then(() => {
+              if (!res.sendEvent) {
```

Comment:
This can be removed no?

Reply:
That is in relation to preventing [events from firing](https://github.com/parse-community/parse-server/pull/6951/files#diff-3fce32ad83d5e4ef0d3cf254e3240b5531dfeebd7af44a76b91119ed05d32a73R265), which was previously achieved by throwing in the trigger. Setting request.sendEvent = false in LQ trigger returns before the events are pushed to the LQ.

Reply:
Since `maybeRunAfterEventTrigger` is the only code that can throw an error this block will be skipped.

Reply:
Sorry @dplewis, I'm not sure what the issue is, what would you like me to change? 😊

Reply:
Throwing an error handles events from firing. `sendEvent` can be removed.

Reply:
Ok, no worries. I added `sendEvent` because if you are using a highly restrictive LQ, you'll end up with afterEvent events filling up your logs, which I'm not sure is desired.

For example, only passing LQ update on "foo" change:

```
Parse.Cloud.afterLiveQueryEvent('TestObject', req => {
      if (req.object.get('foo') != req.original.get('foo')) {
          // if TestObject has a lot of fields, this will throw often
          throw 'Foo was not updated';
      }
});
```

Would log on every single LQ event for TestObject, and pass a `subscription.on('error')` event. The idea behind the sendEvent is a way to stop events firing, without it sending errors to the client, and without it filling up `logLevel: 'error'` logs. However happy to remove it if you see it unnecessary.

Reply:
Oh I see